src/pages/CatalogPage.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import "@/styles/CatalogPage.scss";
+import CatalogItem from "@/components/CatalogItem";
+import { Spinner } from "react-bootstrap";
+import { useParams, useNavigate } from "react-router-dom";
+import MenuInCatalog from "@/components/MenuInCatalog";
+import { useCatalogFetch } from "@/hooks/useCatalogFetch";
+import queryString from "query-string";
+import { APIRouters } from "@/components/fetchWrapper";
+import { toast } from "react-toastify";
+
+const CatalogPage = () => {
+  const params = useParams();
+  const genrePageCatalog: string | undefined = params.name;
+  const urlCatalog: string = queryString.stringifyUrl({
```

Comment:
It's definitely the responsibility not of the Catalog page. Just provide the genre name of param of hook